dom/vehicle.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2022 The Drivers Cooperative. All rights reserved.
+// Use of this source code is governed by a dual
+// license that can be found in the LICENSE file.
+
+package dom
+
+type CarType uint8
+
+const (
+	CarTypeSedan = iota
+	CarTypeHatch
+	CarTypeTwoDoor
+	CarTypeCabrio
+	CarTypeSUV
+	CarTypeVan
+	CarTypeBus
+	CarTypeTruck
+	CarTypeCargoVan
+	CarTypeLimo
+)
```

Comment:
This is okay for a placeholder. We may need to make this configurable by the client; the definition of a "Black Car" or a "SUV" can vary from market to market.